File: Test/InsertTarget.c
#include<stdio.h>
int searchInsert(int* nums, int numsSize, int target){
    int index=0;
    for(int i=0;i<numsSize;i++)
    {
        if(nums[i]<target){
            index++;
        }else{
            return index;
        }
    }
    if(index>0){
        return index;
    }
    else{
        return 0;
    }
}
void main(){
    int arr[4]={1,3,5,6};
    printf("%d",searchInsert(arr,4,0));
}

File: Test/insertTarget.java
public class insertTarget {
    public static void main(String[] args) {
        int nums[]={1,2,3,4};
        System.out.println(searchInsert(nums, 8));
    }
    static int searchInsert(int[] nums, int target) {
    int index=0;
    for(int i=0;i<nums.length;i++)
    {
        if(nums[i]<target){
            index++;
        }else{
            return index;
        }
    }
    if(index>0){
        return index;
    }
    else{
        return 0;
    }
}

}